Corona-Warn-App/src/main/res/values-de/antigen_strings.xml

```diff
@@ -188,17 +188,17 @@
     <!-- XBUT: Create RAT Qr code profile next button-->
     <string name="rat_qr_code_profile_next_button">Weiter</string>
     <!-- XTXT: Create RAT Qr code profile delete item-->
-    <string name="rat_qr_code_profile_delete_item">Schnelltest-Profil löschen</string>
+    <string name="rat_qr_code_profile_delete_item">Schnelltest-Profil entfernen</string>
```

Comment:
I think we changed every `entfernen` into `löschen` intentionally ?

Reply:
See https://github.com/corona-warn-app/cwa-app-android/pull/4357

Reply:
HI, I thought that too... @davidlindercwa @hermesmar  Could you please confirm!?